data/formats-data.js

```diff
@@ -5779,21 +5786,21 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	pumpkaboo: {
-		randomBattleMoves: ["willowisp", "shadowsneak", "destinybond", "synthesis", "seedbomb", "leechseed"],
+		randomBattleMoves: ["trickortreat", "fireblast", "shadowball", "synthesis", "gigadrain", "synthesis"],
 		tier: "LC",
 	},
 	pumpkaboosmall: {
-		randomBattleMoves: ["willowisp", "shadowsneak", "destinybond", "synthesis", "seedbomb"],
+		randomBattleMoves: ["trickortreat", "fireblast", "shadowball", "synthesis", "gigadrain"],
 		unreleasedHidden: true,
 		tier: "LC",
 	},
 	pumpkaboolarge: {
-		randomBattleMoves: ["willowisp", "shadowsneak", "leechseed", "synthesis", "seedbomb"],
+		randomBattleMoves: ["willowisp", "explosion", "destinybond", "synthesis", "bulletseed", "trickroom"],
```

Comment:
+ shadow sneak

Reply:
feels too distracting from the three things i'll want to do with this: tr set, productively kms, or both